event_registration_mail_compose/README.rst

```diff
@@ -0,0 +1,4 @@
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
```

Comment:
not sure moka CI ig generating the readme, probably we have to generate ourself or decide to not generate or keep it empty ??

Reply:
sorry I figure out we are on an OCA repo....